practicals/LLM_prac.ipynb

```diff
@@ -0,0 +1,411 @@
+{
```

Comment:
<p>Consider update as suggested in the first comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2023/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>